apps/pdal.cpp

```diff
@@ -208,17 +215,76 @@ void App::addArgs(ProgramArgs& args)
         m_showOptions);
 }
 
+namespace
+{
```

Comment:
Why not just capture the logPtr into the sig_t function?  It doesn't look like it's directly used anyplace other than main().